src/auro-popover.js

```diff
@@ -9,30 +9,89 @@ import { LitElement, html, css } from "lit-element";
 import "focus-visible/dist/focus-visible.min.js";
 import styleCss from "./style-css.js";
 
-// build the component class
-class AuroPopover extends LitElement {
-  // constructor() {
-  //   super();
-  // }
+import Popover from "./tooltip";
 
-  // function to define props used within the scope of thie component
+class AuroPopover extends LitElement {
+  // function to define props used within the scope of this component
   static get properties() {
     return {
-      cssClass:   { type: String }
+      placement:  { type: String },
+      for:        { type: String },
     };
   }
 
   static get styles() {
     return css`
-      ${styleCss}
+      ${styleCss},
     `;
   }
 
+  patchBuildless() {
+    // patch for buildless environments
+    const code = 'var process = {env: {}};',
+    script = document.createElement('script');
+
+    script.type = 'text/javascript';
+    try {
+      script.appendChild(document.createTextNode(code));
+    } catch (err) {
+      script.text = code;
+    }
+
+    return script;
+  }
+
+  firstUpdated() {
+    // needs to eval before Popover instantiation
+    document.querySelector(`#${this.for}`).parentElement.appendChild(this.patchBuildless());
+
+    const button = document.querySelector(`#${this.for}`),
+     element = this.shadowRoot.querySelector('#tooltip'),
+     hideEvents = [
+      'mouseleave',
+      'blur'
+      ],
+     popper = new Popover(button, element, this.placement),
+     showEvents = [
+      'mouseenter',
+      'focus'
+      ],
+
+    /**
+     * Hides the popover
+     * @returns {Void} Fires an update lifecycle.
+     */
+    toggleHide = function() {
+      element.removeAttribute('data-show');
+      popper.hide();
+    },
+
+    /**
+     * Shows the popover
+     * @returns {Void} Fires an update lifecycle.
+     */
+     toggleShow = function() {
+      element.setAttribute('data-show', '');
+      popper.show();
+    };
+
+    showEvents.forEach((event) => {
+      button.addEventListener(event, toggleShow);
+    });
+
+    hideEvents.forEach((event) => {
+      button.addEventListener(event, toggleHide);
+    });
+
+  }
+
+
   // function that renders the HTML and CSS into  the scope of the component
   render() {
     return html`
-      <div class=${this.cssClass}>
+      <div id="tooltip" class="tooltip auro util_insetLg" role="tooltip">
```

Comment:
for the popover there is a `tooltip` CSS class? Again, I think that the naming needs to be more consistent. 

Also, what is the `auro` selector for?